scripts/generate-release-notes.sh

```diff
@@ -143,6 +143,23 @@ sha1sum of genesis wasm   : $GENESIS_WASM_HASH
 EOF
 fi
 
+# release notes for enclave binary 
+if [ "$2" != "runtime" ] && [ "$2" != "client" ] && [ "$2" != "both"]; then 
```

Comment:
Why can't have an option of `"$2" == "enclave"` ?

Reply:
I kind of tried following the existing pattern in the script, let me elaborate: 
```bash
if [ "$2" != "client" ]; then
  echo "## Runtime" >> "$1"
  ```
  A snippet, wherein `$2` != `client` we build the `Runtime`, So i used the same pattern, wherein if not `client`, `runtime` and `both`, we build the `enclave`. This is not very intuitive imo but would require good refactoring if we were to update existing pattern. 
  CC: @Kailai-Wang

Reply:
What I mean is: why not replace the whole line with `if [ "$2" == "enclave" ]; then` ? Otherwise what if another new parameter coming in, what will happen here?

Reply:
```
if [ "$2" = "enclave" ] || [ "$2" = "all" ]; then 
``` 
Updated to use this